EmotiBit.h

```diff
@@ -37,12 +42,14 @@ class EmotiBit {
 		ISR_CORRECTION_UPDATE,
 		ISR_CORRECTION_TEST,
 		FACTORY_TEST,
+		NVM_CONTROLLER_TEST,
 		length
 	};
 
-	String firmware_version = "1.2.86";
+	String firmware_version = "1.2.87.emotibitMemoryController-1.EmotiBitEda-5";
 	TestingMode testingMode = TestingMode::NONE;
-	const bool DIGITAL_WRITE_DEBUG = false;
+	const bool DIGITAL_WRITE_DEBUG = true;
+	const bool DC_DO_V2 = true;
```

Comment:
Was this set as a class variable for testing? Why would be want this to be a bool if we are permanently moving towards the new protocol?